### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by gabrielilharco**

### CODE REVIEW

**Comment by rwightman** [comment]
@gabrielilharco looking much cleaner

One point of confusion for me, you pushed up a new B/32 to https://huggingface.co/laion/CLIP-ViT-B-32-DataComp.XL-s13B-b90K

And there is also this one https://huggingface.co/laion/CLIP-ViT-B-32-256x256-DataComp-s34B-b86K (that I measured to be 72.7)

However the former one, 13B samples, is that 72.8?

File: README.md [hunk 1]
```diff
@@ -1,49 +1,34 @@
...
-
-As we describe in more detail [below](#why-are-low-accuracy-clip-models-interesting), CLIP models in a medium accuracy regime already allow us to draw conclusions about the robustness of larger CLIP models since the models follow [reliable scaling laws](https://arxiv.org/abs/2107.04649).
-
-This codebase is work in progress, and we invite all to contribute in making it more accessible and useful. In the future, we plan to add support for TPU training and release larger models. We hope this codebase facilitates and promotes further research in contrastive image-text learning. Please submit an issue or send an email if you have any other requests or suggestions.
+Using this codebase, we have trained several models on a variety of data sources and compute budgets, ranging from [small-scale experiments](docs/LOW_ACC.md) to larger runs including models trained on datasets such as
+[LAION-400M](https://arxiv.org/abs/2111.02114), [LAION-2B](https://laion.ai/blog/laion-5b/) and [DataComp-1B](https://arxiv.org/abs/2304.14108).
```
Comment (rom1504) [inline README.md:9]: Let's maybe put the laion5b paper here for consistency

File: README.md [hunk 2]
```diff
@@ -1,49 +1,34 @@
...
+
+| Model    | Training data | Resolution | # of samples seen | ImageNet zero-shot acc. | 
+| -------- | ------- |  ------- |  ------- |  ------- |  
+| ConvNext-Base | LAION-2B  | 256px | 13B | 71.5% |
+| ConvNext-Large | LAION-2B  | 320px | 29B | 76.9% |
+| ConvNext-XXLarge | LAION-2B | 256px | 34B | 79.5% |
+| ViT-B/32  | DataComp-1B  | 256px | 13B | 72.8% |
+| ViT-B/16  | DataComp-1B  | 224px | 13B | 73.5% |
+| ViT-L/14  | LAION-2B  | 224px | 13B | 75.3% |
+| ViT-L/14  | DataComp-1B  | 224px | 13B | 79.2% |
```
Comment (rom1504) [inline README.md:23]: Maybe let's add H too since it's widely used

**Comment by rom1504** [comment]
Nice!

I left a few comments

**Comment by gabrielilharco** [comment]
Thanks @rwightman and @rom1504! Added H-14 to the main table and updated the LAION-5B ref. @rwightman good catch re. the B/32, it should indeed be 34B samples seen for that one. Any thoughts on the 0.1 accuracy difference?

File: README.md [hunk 3]
```diff
@@ -1,49 +1,34 @@
...
+
+
+| Model    | Training data | Resolution | # of samples seen | ImageNet zero-shot acc. | 
+| -------- | ------- |  ------- |  ------- |  ------- |  
+| ConvNext-Base | LAION-2B  | 256px | 13B | 71.5% |
+| ConvNext-Large | LAION-2B  | 320px | 29B | 76.9% |
+| ConvNext-XXLarge | LAION-2B | 256px | 34B | 79.5% |
+| ViT-B/32  | DataComp-1B  | 256px | 34B | 72.8% |
+| ViT-B/16  | DataComp-1B  | 224px | 13B | 73.5% |
+| ViT-L/14  | LAION-2B  | 224px | 13B | 75.3% |
```
Comment (rom1504) [inline README.md:21]: are you sure that's on 13B samples seen ? I thought that was on 32B. Did you check again the reproducibility paper ?

Comment (rwightman) [reply README.md:21]: yeah that l/14 is 32b

Comment (gabrielilharco) [reply README.md]: indeed, updated

**Review by rom1504 — APPROVED** [review]
LGTM let's merge

**Comment by ludwigschmidt** [comment]
Great, thank you @gabrielilharco !

**Pull request merged**